### PR TITLE
Prepare for 8.15.

### PR DESCRIPTION
**Pull request by silene**

### CODE REVIEW

**Comment by Zimmi48** [comment]
@JasonGross I thought you had fixed fiat-parsers with respect to this kind of errors...

@silene There probably is some confusion (the doc saying that the script is run twice is maybe not that clear):
- `dev/tools/update-compat.py --release` must be run before (in a commit that will make it to `v8.14`);
- `dev/tools/update-compat.py --release` must be run after (in the first commit that will not make it to `v8.14`).

**Comment by silene** [comment]
Yes, that is how I understand it too, but the result of running `--release` was rather underwhelming. So, I was not quite sure if it was worth including in the release. Perhaps I am misunderstanding the actual impact of the changes? (Or perhaps this manipulation had already been performed?)

**Comment by Zimmi48** [comment]
One is supposed to remove a compatibility version (to do before releasing), one is supposed to add the next (to do early in the next release cycle). The reason for separating them is that the former can in theory create compatibility issues (in practice, it does very rarely).

**Comment by silene** [comment]
Sure. What I meant that the removed compatibility file is empty. So, the Coq developers have absolutely nothing to gain from its removal (no dead code can be removed as a consequence), yet it annoys the users that were depending on it. That is why I decided to skip the removal for the release.

**Review by Zimmi48 — APPROVED** [review]
OK, now I get it. So this is the first PR for 8.15. In this case, it's ready to go as soon as @JasonGross fixes fiat-parsers (please be quick).

**Comment by JasonGross** [comment]
@Zimmi48 I made it much easier to update fiat-parsers, but have not entirely fixed the issue.  I'll have to think on how to make it more robust to future version changes.  In the meantime, I've updated coq-scripts with https://github.com/JasonGross/coq-scripts/commit/04ea1826d5a1a3ae1db791741eae2699a544032c and updated fiat-parsers and re-started the CI.  If I haven't fixed this issue by the next time this happens, btw, anyone should be able to make the relevant PR on coq-scripts by running
```bash
export V=16; cp compatibility/Coq__master__Compat.v.in compatibility/Coq__8_${V}__Compat.v.in && sed "s/8_$(($V-1))/8_$(($V-1)) 8_$V/g" -i compatibility/generate-compat-files.sh && ./compatibility/generate-compat-files.sh && git add compatibility/Coq__8_${V}__Compat.v{,.in}
```

**Comment by silene** [comment]
The job just failed with the same issue, as far as I can tell: `Makefile:127: Unrecognized Coq version 8.15+alpha`.

**Comment by JasonGross** [comment]
I see "Error: Unable to locate library Coq.Compat.Coq811.". I thought I had fixed that, let me try again

**Comment by JasonGross** [comment]
Oops, I had forgotten to regenerate the company files, should actually be fixed now

**Comment by silene** [comment]
@coqbot merge now

**Comment by Zimmi48** [comment]
I can do it if you want.

**Comment by Zimmi48** [comment]
@coqbot merge now
